sys/phydat/phydat_str.c

```diff
@@ -44,7 +44,7 @@ void phydat_dump(phydat_t *data, uint8_t dim)
         }
         else if ((data->scale > -5) && (data->scale < 0)) {
             char num[8];
-            size_t len = fmt_s16_dfp(num, data->val[i], data->scale * -1);
+            size_t len = fmt_s16_dfp(num, data->val[i], data->scale);
             num[len] = '\0';
             printf("%s", num);
```

Comment:
now I remember: as fmt's `print()` would be easy to use in this line, it is more verbose to some extend in the lines above and below. And either we use `fmt` to print everything or nothing in this file...